components/git/land.js

```diff
@@ -37,6 +37,11 @@ const landOptions = {
     describe: 'Assume "yes" as answer to all prompts and run ' +
     'non-interactively. If an undesirable situation occurs, such as a pull ' +
     'request or commit check fails, then git node land will abort.'
+  },
+  backport: {
```

Comment:
Not blocking, but is it possible to set the default value of this correctly, without the caller having to use `--backport`? It seems to me that it is known to be a backport in most cases because either the target is not master, or the `[backport ]` metadata in the PR subject. Not a big deal, not part of the core functionality, and easy to add later if its useful.

Reply:
We often have release line-specific PRs where there may not be an equivalent commit in master, so we cannot assume it from the branch. Also, only members of backporters/releasers should be landing backports - so I think it will be clearer to keep the option separate/explicit (at least for now).

Reply:
sounds reasonable.